stock_picking_line_sequence/models/stock.py

```diff
@@ -0,0 +1,97 @@
+# Copyright 2017 Camptocamp SA - Damien Crier, Alexandre Fayolle
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# Copyright 2017 Serpent Consulting Services Pvt. Ltd.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+
+    # re-defines the field to change the default
+    sequence = fields.Integer("HiddenSequence", default=9999)
+
+    # displays sequence on the stock moves
+    sequence2 = fields.Integer(
+        "Sequence",
+        help="Shows the sequence in the Stock Move.",
+        related="sequence",
+        readonly=True,
+        store=True,
+    )
+
+    @api.model
+    def create(self, values):
+        move = super().create(values)
+        # We do not reset the sequence if we are copying a complete picking
+        # or creating a backorder
+        if not self.env.context.get("keep_line_sequence", False):
+            move.picking_id._reset_sequence()
+        return move
+
+
+class StockMoveLine(models.Model):
+    _inherit = "stock.move.line"
+
+    def _get_aggregated_product_quantities(self, **kwargs):
+        def get_aggregated_properties(move_line=False, move=False):
+            move = move or move_line.move_id
+            uom = move.product_uom or move_line.product_uom_id
+            name = move.product_id.display_name
+            description = move.description_picking
+            if description == name or description == move.product_id.name:
+                description = False
+            product = move.product_id
+            line_key = f'{product.id}_{product.display_name}_{description or ""}_{uom.id}_{move.product_packaging_id or ""}'
+
+            return line_key
```

Comment:
Well spotted that the line had to change in order to be in sync with https://github.com/odoo/odoo/blob/49d25c5/addons/stock/models/stock_move_line.py#L778-L794, but is there any reason why we can't just call that method and use the line_key that that method returns in its result dictionary?